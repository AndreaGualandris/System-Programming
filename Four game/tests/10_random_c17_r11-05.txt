10
5
14
15
5
4
11
6
11
10
14
15
12
5
15
6
1
12
8
11
11
6
1
14
13
13
15
2
6
5
10
15
11
0
6
8
2
3
0
6
15
5
7
5
0
2
5
9
1
13
0
15
14
6
14
13
5
12
6
4
4
0
8
11
14
6
13
16
2
10
0
16
16
3
5
3
13
1
3
11
2
11
2
6
10
0
1
14
3
12
13
5
14
15
14
15
10
11
9
8
15
0
3
3
5
4
15
5
4
1
2
14
16
14
14
10
0
2
10
9
0
7
2
1
4
14
3
2
3
14
1
12
6
5
6
12
14
9
11
15
8
7
9
5
16
0
7
5
15
0
14
6
16
4
12
0
7
1
4
10
16
7
15
0
16
10
12
3
10
2
10
5
1
9
16
14
6
1
6
2
12
7
13
0
2
1
3
