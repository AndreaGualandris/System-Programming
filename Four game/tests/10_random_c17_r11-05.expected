-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
          o
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
          o      
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x    o      
-----------------
01234567890123456
Player 'o' to move:
              o
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x    o   o  
-----------------
01234567890123456
Player 'x' to move:
               x
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x    o   ox 
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o           
     x    o   ox 
-----------------
01234567890123456
Player 'x' to move:
    x
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o           
    xx    o   ox 
-----------------
01234567890123456
Player 'o' to move:
           o
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o           
    xx    oo  ox 
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o           
    xxx   oo  ox 
-----------------
01234567890123456
Player 'o' to move:
           o
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o     o     
    xxx   oo  ox 
-----------------
01234567890123456
Player 'x' to move:
          x
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o    xo     
    xxx   oo  ox 
-----------------
01234567890123456
Player 'o' to move:
              o
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o    xo  o  
    xxx   oo  ox 
-----------------
01234567890123456
Player 'x' to move:
               x
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o    xo  ox 
    xxx   oo  ox 
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     o    xo  ox 
    xxx   ooo ox 
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x           
     o    xo  ox 
    xxx   ooo ox 
-----------------
01234567890123456
Player 'o' to move:
               o
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x         o 
     o    xo  ox 
    xxx   ooo ox 
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x         o 
     ox   xo  ox 
    xxx   ooo ox 
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x         o 
     ox   xo  ox 
 o  xxx   ooo ox 
-----------------
01234567890123456
Player 'x' to move:
            x
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x         o 
     ox   xox ox 
 o  xxx   ooo ox 
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x         o 
     ox   xox ox 
 o  xxx o ooo ox 
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x     x   o 
     ox   xox ox 
 o  xxx o ooo ox 
-----------------
01234567890123456
Player 'o' to move:
           o
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
           o     
     x     x   o 
     ox   xox ox 
 o  xxx o ooo ox 
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
           o     
     xx    x   o 
     ox   xox ox 
 o  xxx o ooo ox 
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
           o     
     xx    x   o 
 o   ox   xox ox 
 o  xxx o ooo ox 
-----------------
01234567890123456
Player 'x' to move:
              x
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
           o     
     xx    x  xo 
 o   ox   xox ox 
 o  xxx o ooo ox 
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
           o     
     xx    x  xo 
 o   ox   xox ox 
 o  xxx o ooooox 
-----------------
01234567890123456
Player 'o' wins!
